src/wp-includes/class-wp-query.php

```diff
@@ -2020,7 +2020,7 @@ public function get_posts() {
 		}
 
 		if ( isset( $q['page'] ) ) {
-			$q['page'] = trim( $q['page'], '/' );
+			$q['page'] = is_scalar( $q['page'] ) ? trim( $q['page'], '/' ) : '';
 			$q['page'] = absint( $q['page'] );
```

Comment:
Querion: If `$q['page']` is not scalar then it is wroth to check `absint( $q['page'] )`. WDYT?

Reply:
Is there any overhead to `absint` if not, it does provide an additional safety check. Happy to  change it, but would be curious what others think.